### PR TITLE
Services

### PR DESCRIPTION
**Pull request by cynexit**
Added first working and tested cuckoo service, the whole package can now be deployed but should still be considered to be in a early development stage.
Detailed documentation on how to set everything up will be added soon.

### CODE REVIEW

File: services/cuckoo/processing.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+				})
+			}
+		}
+	}
+
+	return res
+}
+
+/*
+// support for dropped files will be added later
```
Comment (webstergd) [inline services/cuckoo/processing.go:161]: Should maybe be a special flag with a huge warning about this. This can bloat your system really fast.

**Comment by webstergd** [comment]
Do we want to Dockerize the services? I can do that later if we want to go that route.

**Comment by cynexit** [comment]
Services should be dockerized, I added the necessary file and made some adjustments to travis so it doesn't try to build the services but just the main project.

**Comment by webstergd** [comment]
Need to tweak docker and readme.md. However, we can do that will a follow on pull. I think it is more important to get this accepted for now.

**Pull request merged**